test/lib/BedrockTester.h

```diff
@@ -53,6 +53,8 @@ class BedrockTester {
     // but not remove args. Any args specified here are added or replaced into the existing set.
     void updateArgs(const map<string, string> args);
 
+    string getArg(const string& arg) const;
```

Comment:
Yes, essentially.

Reply:
Should we rename it for clarity (and the other methods related to this one)? 😁

Reply:
Meh, it's named `args` or `_args` everywhere else in bedrock too, so, it's sort of more consistent this way. I can change it if you insist, but I'm not particularly motivated otherwise. Let me know.

Reply:
NAB 😄